Source/Core/VideoCommon/NetPlayChatUI.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <array>
+#include <deque>
+#include <functional>
+#include <memory>
+#include <string>
+#include <utility>
+
+class NetPlayChatUI
+{
+public:
+  explicit NetPlayChatUI(std::function<void(const std::string&)> callback);
+  void Display();
+  void AppendChat(const std::string& message, const std::string& color);
+
+private:
+  char message_buf[160] = {};
```

Comment:
```suggestion
  char m_message_buf[160] = {};
```

Reply:
I'd like to add that this buffer should probably be larger, as 160 characters isn't a lot if someone is typing out a particularly long sentence. 1024 would probably be sufficient (I like powers of two).